### PR TITLE
release 0.2.4

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation

https://issues.jboss.org/browse/INTLY-3047

## What

Release 0.2.4 with fix.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.2.4] - 2019-08-27
+
+### Fix
```
Comment (craicoverflow) [inline CHANGELOG.md]: Should be "Fixed" according to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

**Review by odra — APPROVED** [review]

**Review by craicoverflow — CHANGES_REQUESTED** [review]

**Comment by camilamacedo86** [comment]
Hi @craicoverflow, 

what is the change that you requested here?

**Comment by craicoverflow** [comment]
> Hi @craicoverflow,
> 
> what is the change that you requested here?

I left a comment on the line that should be fixed.

https://github.com/aerogear/mobile-developer-console-operator/pull/53#discussion_r317995626

File: CHANGELOG.md (hunk 1 above)
Comment (camilamacedo86) [reply CHANGELOG.md]: Done

**Review by craicoverflow — APPROVED** [review]

**Pull request merged**